src/components/SearchGridFilter.vue

```diff
@@ -129,4 +129,9 @@ export default {
   }
 }
 
+#content-wrapper {
```

Comment:
You can add this to the `__visible` [style](https://github.com/creativecommons/cccatalog-frontend/blob/527df3f30019e8bb2b0e9a5e901dce8cec9ffa62/src/components/SearchGridFilter.vue#L126). No need to add an id here

Reply:
Done @brenoferreira